apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js

```diff
@@ -5,8 +5,14 @@ import React from 'react';
 import _ from 'lodash';
 import {expect} from 'chai';
 import {mount} from 'enzyme';
+import sinon from 'sinon';
 
 describe("DetailViewContents", () => {
+  // We aren't testing any of the responses of the workshop selector control, so just
+  // return success for all requests for the time being to suppress the warning message
+  // in the test output
+  sinon.fakeServer.create();
```

Comment:
note - the comment isn't quite accurate. This won't return anything (success or otherwise). It creates a stub server, but unless you set a response this will simply not return from the API calls. Therefore it won't hit your done callbacks.

Reply:
okay - I don't have any done callbacks at the time so I'm fine with the fake server. I'll just fix the comment